app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordPrescribedDrug.kt

```diff
@@ -0,0 +1,30 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord
+
+import androidx.room.Dao
+import androidx.room.Entity
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy.REPLACE
+import androidx.room.PrimaryKey
+import androidx.room.Query
+import java.util.UUID
+
+@Entity
+data class TeleconsultRecordPrescribedDrug(
+    @PrimaryKey
+    val teleconsultRecordId: UUID,
+    val prescribedDrugUuid: UUID
```

Comment:
Shouldn't this be a composite primary key based of both the IDs? Since a single teleconsult record might have multiple prescriptions, saving a list of teleconsult prescribed drug records will end up only saving one since each one will replace the previous one.